cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -390,6 +392,61 @@ This ensures that the next time the MapReduce job runs, it processes only the ne
     partitionCommitter.onFinish(succeeded);
   }
 
+Processing using a Generic
```

Comment:
I'm not crazy about this title, as *Generic* really refers to something else...any suggestions?

What about "Processing using a PartitionConsumer"?